pkg/plugins/backendplugin/contracts.go

```diff
@@ -91,3 +91,46 @@ type CallResourceResult struct {
 	Headers map[string][]string
 	Body    []byte
 }
+
+type CallResourceResultStream interface {
```

Comment:
AFAICT this shouldn't be exported, since it isn't used outside the package (I could build with it unexported).